src/atl_store.rs

```diff
@@ -0,0 +1,114 @@
+use dashmap::DashMap;
+use itertools::Itertools;
+use solana_address_lookup_table_program::state::AddressLookupTable;
+use solana_rpc_client::nonblocking::rpc_client::RpcClient;
+use solana_sdk::{pubkey::Pubkey, slot_hashes::SlotHashes, slot_history::Slot, commitment_config::CommitmentConfig};
+use std::sync::Arc;
+
+use crate::block_info::TransactionAccount;
+
+pub struct ATLStore {
+    rpc_client: Arc<RpcClient>,
+    pub map: Arc<DashMap<String, Vec<u8>>>,
```

Comment:
should control somehow the size of this map; maybe log the size if it exceeds a certain size

Reply:
Nah should be ok for now. There are not many ATL accounts.